source/module.c

```diff
@@ -841,6 +835,13 @@ const char *aws_napi_status_to_str(napi_status status) {
         case napi_bigint_expected:
             reason = "napi_bigint_expected";
             break;
+/* TODO: The enum `napi_no_external_buffers_allowed` is introduced in node14.
```

Comment:
Is there a NAPI_VERSION value we could use here instead of commenting it out?

Reply:
Looking into the napi header files, I didn't see the NAPI_VERSION used for the enum. Meanwhile, I also didnt see any NAPI_VERSION ties to `napi_queue_full`, however, we did use NAPI_VERSION to opt in the enum value. I'm curious how could we determin which NAPI_VERSION the enum is introduced?